content/influxdb/cloud-iox/query-data/tools/grafana.md

```diff
@@ -51,30 +51,45 @@ Because Grafana Flight SQL Plugin is a custom plugin, you can't use it with Graf
 For more information, see [Find and Use Plugins in the Grafana Cloud documentation](https://grafana.com/docs/grafana-cloud/fundamentals/find-and-use-plugins/)
 {{% /warn %}}
 
-## Download the Grafana Flight SQL plugin
-
-Download the latest release from [influxdata/grafana-flightsql-datasource releases](https://github.com/influxdata/grafana-flightsql-datasource/releases).
+## Create a custom plugins director
 
-{{% warn %}}
-The Grafana Flight SQL plugin is experimental and subject to change.
-{{% /warn %}}
+Create a **custom plugins directory** to store the Flight SQL plugin in and
+navigate into the directory.
+_The custom plugins directory can be anywhere in your filesystem that Grafana can access._
 
+{{% code-callout "/custom/plugins/directory" %}}
 ```sh
-curl -L https://github.com/influxdata/grafana-flightsql-datasource/releases/download/v0.1.9/influxdata-flightsql-datasource-0.1.9.zip --output influxdata-flightsql-datasource.zip
+mkdir -p /custom/plugins/directory/ && cd $_
 ```
+{{% /code-callout %}}
 
-<span id="custom-grafana-plugins-directory"></span>
+## Download the Grafana Flight SQL plugin
 
-## Extract the Flight SQL plugin
+Download the latest [Grafana Flight SQL plugin (influxdata/grafana-flightsql-datasource)](https://github.com/influxdata/grafana-flightsql-datasource/releases).
```

Comment:
Remove this and add the link below the shell script instruction to avoid confusion? E.g. 

View [link] releases...